src/api/test-helpers/declaration.ts

```diff
@@ -45,9 +45,35 @@ const get = async (context: Context, declarationId: string) => {
   }
 };
 
+/**
+ * Update declaration test helper
+ * Update a declaration by ID
+ * @param {Context} context: KeystoneJS context API
+ * @param {String} declarationId: Declaration ID
+ * @param {ApplicationDeclaration} data: Declaration data
+ * @returns {Promise<ApplicationDeclaration>} Declaration
+ */
+const update = async (context: Context, declarationId: string, data = {}) => {
+  try {
+    console.info('Updating a declaration by ID (test helpers)');
+
+    const declaration = await context.db.Declaration.updateOne({
+      where: { id: declarationId },
+      data,
+    });
+
+    return declaration;
+  } catch (error) {
+    console.error(error);
```

Comment:
Add some text please.

Reply:
Agree this would be nice to have.

We have 37 test helper files with multiple functions containing this; Do not want to bloat this PR or become misaligned - have created EMS-4099 to tackle this in a different PR.

Reply:
I would just note that these files are used purely for unit testing purposes.

Reply:
As long as it will rectified in this PR EMS-4099 I am happy to approve for now.